web3/wallets.js

```diff
@@ -624,12 +679,30 @@ exports.getPrices = async () => {
         }
 
             var finalMap = {}
+            var idcrypto =[]
             for (var i = 0; i < priceMap.length; i++) {
+            
+              idcrypto.push(priceMap[i].id.toString())
                 finalMap[priceMap[i].symbol] = priceMap[i]
+        
                 delete finalMap[priceMap[i].symbol].symbol
             }
+            
+    try{
+        networksContract= await   getNetworkByToken( idcrypto.join(','))
+        for (var i = 0; i < idcrypto?.length; i++) {
```

Comment:
It seems like there are quite a few loop methods within the getPrice function, which might be causing a performance hit, especially when dealing with large data sets. Could you provide more context about the function and the data it's handling? With more information, we can discuss potential ways to refactor the function to make it more efficient. Specifically, we might look at alternatives to the loop methods being used and determine whether we can minimize their usage, or consider different data structures that could provide the same functionality but with better performance.